source/includes/api/_areas_v2.md

```diff
@@ -326,46 +393,25 @@ curl -X PATCH https://api.resourcewatch.org/v2/area/:id
 }
 ```
 
-Use this endpoint to update an existing area. This endpoint requires authentication and, in order to PATCH an area, you need to be either the owner of the area or be an ADMIN user.
-
-This endpoint supports the following request body parameters:
-
-Field                |             Description                                                                            | Type    | Example    |
--------------------- | :------------------------------------------------------------------------------------------------: | ------: | ---------: |
-name                 | The name of the area being created.                                                                | String  | 'Example'  |
-image                | Image associated with the dataset - in GET areas, this attribute will have the URL for the image.  | String  | https://www.google.com/example.jpg |
-application          | The application to which this area is associated with. Defaults to 'gfw'.                          | String  | 'gfw'      |
-language             | The language of this area. Defaults to 'en'.                                                       | String  | 'es'       |
-geostore             | An ID of a geostore to which this area relates to.                                                 | String  | '123'      |
-public               | If the area is public or not. Defaults to false.                                                   | Boolean | true       |
-fireAlerts           | If the area is intended to subscribe to fire alerts. Defaults to false.                            | Boolean | true       |
-deforestationAlerts  | If the area is intended to subscribe to deforestation alerts. Defaults to false.                   | Boolean | true       |
-monthlySummary       | If the area is intended to subscribe to monthly summaries. Defaults to false.                      | Boolean | true       |
-email                | Email to be provided to the subscription.                                                          | String  | youremail@resourcewatch.org |
-webhookUrl           | Webhook URL to be provided to the subscription (only used in case the email is not set).           | String  | https://www.google.com/ |
-status               | The status of the area - either 'saved' or 'pending'. Read-only attribute.                         | String  | 'saved'    |
-subscriptionId       | The ID of the subscription associated with this area. Read-only attribute.                         | String  | 5e4d7c47dd8fa31290d548ae |
+Use this endpoint to update an existing area. For a detailed description of each field that can be provided in the body of the request, check out the [Area reference](#area-reference) section.
 
-### Email notification
+Keep in mind that you should provide one of the following when updating an area:
```

Comment:
Should or must?

Reply:
Must, otherwise it will return 401 Unauthorized - thanks for pointing out :+1: